### PR TITLE
Increase fs inotify resources for e2e tests

### PR DESCRIPTION
**Pull request by mitalipaygude**
*Issue #, if available:*

*Description of changes:*
The docker e2e tests are failing because of kubelet errors regarding too many open files.
It is a known issue and based on kind documentation [here](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files), this pr fixes the issue.

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by vivek-koppuru — APPROVED** [review]

File: internal/test/e2e/setup.go [hunk 1]
```diff
@@ -153,6 +159,17 @@ func (e *E2ESession) setup(regex string) error {
 	return nil
 }
 
+func (e *E2ESession) updateFSInotifyResources() error {
+	command := fmt.Sprintf("sudo sysctl fs.inotify.max_user_watches=%v && sudo sysctl fs.inotify.max_user_instances=%v", maxUserWatches, maxUserInstances)
```
Comment (jaxesn) [inline internal/test/e2e/setup.go:163]: Can we call sudo via ssm?  Not sure if we are already or not...

Comment (mitalipaygude) [reply internal/test/e2e/setup.go:163]: we did [here](https://github.com/aws/eks-anywhere/blob/main/internal/test/e2e/registryMirror.go#L37) so should be fine

**Review by jaxesn — APPROVED** [review]

**Pull request merged**